docs/guidelines.md

```diff
@@ -113,7 +113,30 @@ ngeo.exampleDirective = function(…) {
       // …
     },
     controllerAs: 'ctrl',
+    // …
+  });
+```
+
+But if you bind your directive to the controller and you declare your attribute
+as a two ways binding attrribute, you can use the `.` notation. See the example
```

Comment:
Thanks @ger-benjamin for having a stab at this section. I discussed it with @sbrunner and we wanted to clarify it.

What you propose is almost the good way. You need to change it to:
```
/**
  @type {ol.Map}
  @export
*/
this.m;
```

The above code `declares` a typed variable called `m` and asks closure not to rename it.
It is then possible to access the variable using the dotted notation AND closure will check all access are valid.

Now, why you code worked?
Closure rename the variables even if their size is optimal (m is already 1 character).
So the only explanation I see would be that there exists somewhere a type with an attribute named `m`.
That is a bit surprising though.

@ger-benjamin, could you change to the code above and remove the part about using the bracket notation?